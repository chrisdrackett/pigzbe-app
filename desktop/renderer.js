import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
