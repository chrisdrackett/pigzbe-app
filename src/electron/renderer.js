import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
