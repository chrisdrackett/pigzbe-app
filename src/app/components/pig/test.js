import React from 'react';
import renderer from 'react-test-renderer';
import Pig from './';

describe('Pig', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<Pig/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
