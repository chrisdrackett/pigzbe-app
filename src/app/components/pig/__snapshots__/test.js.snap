// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pig renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "flex": 1,
      "justifyContent": "flex-end",
      "position": "absolute",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/app/components/pig/pig.png",
      }
    }
    style={
      Object {
        "height": 35,
        "width": 81,
      }
    }
  />
</View>
`;
